.php-cs-fixer.dist.php

```diff
@@ -44,6 +44,10 @@
         'header_comment' => ['header' => $header],
         'modernize_strpos' => true, // needs PHP 8+ or polyfill
         'no_useless_concat_operator' => false, // TODO switch back on when the `src/Console/Application.php` no longer needs the concat
+        'numeric_literal_separator' => [
+            'override_existing' => false,
+            'strategy' => 'use_separator',
```

Comment:
As I wrote [here](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/6761#discussion_r1447943407) I don't agree with this default, I believe it should be as is now (`no_separator`).

In terms of applying it in the codebase, I am soft-against. Personally I don't like this convention and don't need it, but I can live with it if we decide to include it in our ruleset 🙂.

Reply:
this PR doesn't change the defaults - only configuring in-project usage.
we conclude defaults in linked PR 👍🏻 

for this PR, i am proposing to use it for project, but it could e my habit from other languages, happy to hear more opinions and not pushing for it myself.

Reply:
I have no preference but maybe it's because I'm not used to such convention. Maybe I'll like it after using it for a while so I'd give it try.

Reply:
we got soft-no from Wirone and soft-yes to try it out from Julien.

Let me merge as-is, without adding to ruleset. Let's see how we deal with it on our own codebase, if we won't like it on long-run, we can revert it. If we would like it, we can consider recommend it via ruleset.